src/proposals/Addresses.sol

```diff
@@ -584,18 +585,40 @@ contract Addresses is Test, ChainIds {
             0xB88BAc61a4Ca37C43a3725912B1f472c9A5bc061
         );
 
+        /// https://docs.chain.link/data-feeds/price-feeds/addresses?network=base&page=1&search=reth
+        /// https://basescan.org/address/0xf397bF97280B488cA19ee3093E81C0a77F02e9a5
+        _addAddress(
+            "CHAINLINK_RETH_ETH",
+            baseChainId,
+            0xf397bF97280B488cA19ee3093E81C0a77F02e9a5
```

Comment:
Confirmed this contract is deployed and returning a reasonable `latestAnswer` of `1094571593701889800`